tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_ops.py

```diff
@@ -443,7 +464,8 @@ def embedding_lookup(
       params: A dynamic_embedding.Variable instance.
       ids: A tensor with any shape as same dtype of params.key_dtype.
       partition_strategy: No used, for API compatiblity with `nn.emedding_lookup`.
-      name: A name for the operation (optional).
+      name: A name for the operation. Name is optional in graph mode and required
+        in eager mode.
```

Comment:
I think a check is needed to make sure the `name` is set in eager mode.

Reply:
It's checked when creating TW. I moved it to the front of func entry now.